src/main/java/io/anserini/index/IndexUtils.java

```diff
@@ -54,9 +72,22 @@
     @Option(name = "-printDocvector", metaVar = "docid", usage = "prints the document vector of a document")
     String docvectorDocid;
 
+    @Option(name = "-dumpAllDocids", usage = "dumps all Docids in sorted order. For non-tweets collection the order is " +
```

Comment:
So... I'm not sure we need to have the file above?
It's just a repeat of what's here, no?

Reply:
I think the `index-utils.md` is useful in the sense that we could have an entry in the `README.md` so that people know about this utility?

Reply:
Sure, but no need to repeat the options? This forces us to keep the docs in sync with the code... over time, it'll drift apart...